### PR TITLE
[tests-only]Add new issue tag for spaces

### LINKED ISSUE #39699: Tag "personal spaces" test scenarios
We are adding test scenarios (examples) for the "personal space" style of `dav` endpoint. Those have an extra examples table for `dav_version` set to `spaces`. See PR #39686 for an example.

We are running those tests in oCIS now, which is good.

We are starting to run them in `cs3org/reva`. But we want to run them (for now) just in the `edge` branch and not in the `master` branch of reva. In future we might want to switch them on and off in different environments.

Add a "feature tag" to all the scenarios with `dav_version` set to `spaces`. That will let us easily filter/skip those scenarios in CI.

I suggest that we tag them `@personalSpace`

### PR DESCRIPTION
**Pull request by amrita-shrestha**
## Description
New **`@personalSpace`** issue tag added to all the scenarios with dav_version set to spaces

## Related Issue

- Fixes https://github.com/owncloud/core/issues/39699

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

File: tests/acceptance/features/apiFavorites/favorites.feature [hunk 1]
```diff
@@ -182,8 +182,12 @@ Feature: favorite
       | dav_version |
       | old         |
       | new         |
-      | spaces      |
 
+    @personalSpace
+    Examples:
+      | dav_version |
+      | spaces      |
+skipOnOcV10
```
Comment (phil-davis) [inline tests/acceptance/features/apiFavorites/favorites.feature]: random text needs removing

File: tests/acceptance/features/apiFavorites/favorites.feature [hunk 2]
```diff
@@ -197,7 +201,7 @@ Feature: favorite
     And user "Alice" has favorited element "/subfolder/textfile0.txt"
     And user "Alice" has favorited element "/subfolder/textfile1.txt"
     And user "Alice" has favorited element "/subfolder/textfile2.txt"
-    And user "Alice" has favorited element "/subfolder/textfile3.txt"
+    And user "Alice" has favorited e      | spaces      |lement "/subfolder/textfile3.txt"
```
Comment (phil-davis) [inline tests/acceptance/features/apiFavorites/favorites.feature]: random text needs removing

File: tests/acceptance/features/apiFavorites/favorites.feature [hunk 3]
```diff
@@ -211,6 +215,13 @@ Feature: favorite
       | dav_version |
       | old         |
       | new         |
+
+    @personalSpace
+
+
+
```
Comment (phil-davis) [inline tests/acceptance/features/apiFavorites/favorites.feature]: please delete empty lines

File: tests/acceptance/features/apiFavorites/favorites.feature [hunk 4]
```diff
@@ -254,7 +265,7 @@ Feature: favorite
       | old         |
       | new         |
 
-    @skipOnOcV10
+    @personalSpace
```
Comment (phil-davis) [inline tests/acceptance/features/apiFavorites/favorites.feature]:
```suggestion
    @skipOnOcV10 @personalSpace
```

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiAuthWebDav-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34358/47/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiAuth-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34358/45/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiComments-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34358/49/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiProvisioningGroups-v1-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34358/54/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiCapabilities-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34358/48/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiProvisioning-v2-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34358/53/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiFavorites-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34358/50/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiProvisioningGroups-v2-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34358/55/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiAuthOcs-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34358/46/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>webUIUpload-chrome-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34358/149/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>webUIProxySmoke-3-3-chrome-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34358/165/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>webUISharingExt2-latest-chrome-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34358/158/1

**Review by phil-davis — APPROVED** [review]

**Comment by phil-davis** [comment]
@amrita-shrestha please update the cs3org/reva master PR so that it bumps the core commit id to the very latest.
And adjust the filter-tags so that it skips `@personalSpace`

**Pull request merged**